packages/component/src/Activity/ActivityFeedback.tsx

```diff
@@ -1,23 +1,56 @@
 import { hooks } from 'botframework-webchat-api';
 import { getOrgSchemaMessage, OrgSchemaAction, parseAction, WebChatActivity } from 'botframework-webchat-core';
-import cx from 'classnames';
-import React, { memo, useMemo } from 'react';
+import classNames from 'classnames';
+import React, { memo, useCallback, useMemo, useRef, useState } from 'react';
+import { defaultFeedbackEntities } from './private/DefaultFeedbackEntities';
+import { isDefaultFeedbackActivity } from './private/isDefaultFeedbackActivity';
 
 import Feedback from './private/Feedback';
 import dereferenceBlankNodes from '../Utils/JSONLinkedData/dereferenceBlankNodes';
+import FeedbackForm, { FeedbackType } from './private/FeedbackForm';
+import { useStyleToEmotionObject } from '../hooks/internal/styleToEmotionObject';
 
 const { useStyleOptions } = hooks;
 
 type ActivityFeedbackProps = Readonly<{
   activity: WebChatActivity;
 }>;
 
+const ROOT_STYLE = {
+  '&.webchat__feedback-form__root__container': {
```

Comment:
BEM: `webchat__block__element--modifier` in kebab-case.

- B: `webchat__feedback-form`
- BE: `webchat__feedback-form__root`
- BEM: `webchat__feedback-form__submit-button--submitted`
   - No modifier for :hover, :active, :focus, etc.
- BM: `webchat__feedback-form--submitted`

But no: `BEE`, `BEME`, `BBE`